tfhe/src/integer/compression_keys.rs

```diff
@@ -0,0 +1,56 @@
+use super::ClientKey;
+use crate::integer::backward_compatibility::list_compression::*;
+use serde::{Deserialize, Serialize};
+use tfhe_versionable::Versionize;
+
+#[derive(Clone, Debug, Serialize, Deserialize, Versionize)]
+#[versionize(CompressionPrivateKeysVersions)]
+pub struct CompressionPrivateKeys(pub crate::shortint::list_compression::CompressionPrivateKeys);
```

Comment:
this is not consistent with other keys, they are struct with named fields, see e.g. tfhe/src/integer/server_key/mod.rs::36

Reply:
Replaced by a field named `key`